src/content/docs/docs/resources/op-wallets.mdx

```diff
@@ -3,18 +3,68 @@ title: Web Monetization-enabled wallets
 next: false
 ---
 
-import { Tabs, TabItem } from '@astrojs/starlight/components'
+import { Icon } from '@astrojs/starlight/components';
 import { LinkOut } from '@interledger/docs-design-system'
 
-A Web Monetization-enabled wallet is required to send and/or receive payments. The wallet providers on this page are set up to support Web Monetization.
+*Last updated: October 22, 2024*
 
-Be sure to select a provider that is available in your **region** and supports your preferred **currency**.
+The following wallet providers support sending and receiving Web Monetization payments. 
 
-Your provider will assign you a payment address, called either a wallet address or a payment pointer.
+<div style="border: 1px solid #357873; padding: 10px;">
+<Icon name="information" color="#357873" size="2rem" class="inline-icon" /> Be sure to select a provider that is available in your **region** and supports your preferred **currency**.
+</div>
 
-## Wallet providers
+Your provider will assign you a payment address, called either a wallet address or a payment pointer. Your payment address uniquely identifies your wallet account with your provider and can be shared publicly--similar to how your email address identifies your email account and can be shared publicly.
 
-| Provider | Payment Address Label |
-| --------------- | --------------- |
-| <LinkOut href="https://wallet.fynbos.app">Fynbos</LinkOut> | Wallet address |
-| <LinkOut href="https://gatehub.net">GateHub</LinkOut> | Payment pointer |
\ No newline at end of file
+## <LinkOut href="https://wallet.fynbos.app/">Fynbos</LinkOut>
+
+ <table style="font-size: 14px; border: 1px solid #D3D3D3;">
+  <tr>
+    <th style="background-color: #57ACDC; border-right: 1px solid #D3D3D3; border-bottom: 1px solid #D3D3D3; padding-left: 10px;">Supported regions</th>
+    <td style="border-bottom: 1px solid #D3D3D3;">Canada, South Africa, United States</td>
```

Comment:
Can we move these styles to custom.css?

Reply:
We can do this in the new version of the web monetization docs (which go beyond the extension docs)